### DISCUSSION TITLE
Discord Booster Sync

### DISCUSSION THREAD

**Discussion by JXYZP-G** [gameserverapp/Platform#388] [Suggestions]
**Discord Nitro Perks**
I Don't know if its possible to make the discord bot check who has boosted the server and bring that in sync with your sever so they obtain a custom role that's automatically created by GSA when Boosting the server. Same as "Server Booster" Role in discord.

Features:
- Discord Nitro Sync

There are probably more functionality's but i forgot them if any of you have more ideas that would fit right please reply below!

**Comment by mvaessen** [comment]
duplicate
